qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -41,7 +41,7 @@ class QSVC(SVC, SerializableModelMixin):
         qsvc.predict(sample_test)
     """
 
-    def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwargs):
+    def __init__(self, *args, quantum_kernel: Optional[BaseKernel] = None, **kwargs):
         """
         Args:
             quantum_kernel: QuantumKernel to be used for classification.
```

Comment:
```suggestion
            quantum_kernel: Quantum kernel to be used for classification.
```

Reply:
Maybe you want to create a different issue/PR to address this but a little but further below (cannot directly comment at that point) it has

>f"Positional arguments ({args}) are deprecated as of version 0.3.0 and "

Since that was more 6 months ago and upcoming 0.5.0 will be 2 releases it seems time to do this.

Reply:
#483 is to address this comment, created recently. This PR is getting heavier and heavier.

Reply:
Ah ok, you already have an issue to address that. (Since the deprecation removals sometimes get overlooked I figured I'd mention it).